src/wrapper.cc

```diff
@@ -127,8 +128,10 @@ Dart_CObject CallbackArgBuilder(int num, ...) {
   void *request_buffer = malloc(sizeof(uint64_t) * num);
   uint64_t *buf = reinterpret_cast<uint64_t *>(request_buffer);
 
+  // uintptr_r will get implicitly casted to uint64_t. So, in a 32 bit machine,

```

Comment:
re: 32 bit machine
You can run things in 32 bit mode on 64 bit machines.

Document why this is important: because its read in Dart as a 64 bit integer.

Reply:
Changed. Is the new one good?